src/CommandingOfficers/Meridian.java

```diff
@@ -64,7 +64,7 @@ public Commander create(GameScenario.GameRules rules)
   final VehicularCharge myVehicularCharge = new VehicularCharge(this);
   private static final int POST_REFRESH_STAT_ADJUSTMENT = -30;
   final UnitModel tank, arty;
-  final UnitActionFactory tankAction, artyAction;
+  final UnitActionFactory tankAction, artyAction; // Has to be one per CO, so units get the same instance over multiple action searches (mostly for the AI)
```

Comment:
Asplain. Would one static global instance not satisfy that requirement? What other alternative was there besides that and one per CO?

Reply:
I initially created the action type instance any time the modifier was invoked. (You can see that in the `DeployableCommander` diff on the last commit)

I worry about how a static/global instance would work with savegames, though I guess I could do the thing I did with terrain and make it "deserialize" by pointing to the static instance. Having it in the CO is a small cost of memory for a simpler implementation, though.
...I guess I could test it, but (de)serialization is so full of caveats and black magic that I don't like touching it if I don't have to.